README.adoc

```diff
@@ -33,11 +33,7 @@ Once running they can be executed with:
 
 [source,bash]
 ----
-./gradlew :spring-credhub-integration-tests:test -PintegrationTests
-----
-[source,bash]
-----
-./gradlew :spring-credhub-reactive-integration-tests:test -PintegrationTests
+./gradlew build -PintegrationTests --no-parallel
```

Comment:
Why `--no-parallel`?

Reply:
Integration tests fail when running in parallel. Since the whole build time, including integration tests and running sequentially is still a minute decided not to think very much about it.